keyboards/mechlovin/zed65/910/info.json

```diff
@@ -0,0 +1,351 @@
+{
+    "manufacturer": "Mechlovin Studio",
+    "url": "https://mechlovin.studio/", 
+    "maintainer": "mechlovin",
+    "keyboard_name": "Zed65 910 Rev1",
+    "usb": {
+        "vid": "0x4D4C",
+        "pid": "0x6505",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "backlight": false,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "rows": ["B15", "B14", "B13", "B12", "A1"],
+        "cols": ["B11", "B10", "B2", "B1", "B0", "A6", "A5", "A4", "A3", "C13", "B7", "B6", "B5", "B4", "B3"]
+    },
+    "indicators": {
+        "caps_lock": "B9",
+        "on_state": 0
+    },
+    "diode_direction": "COL2ROW",
+    "rgblight": {
+        "led_count": 4,
+        "sleep": true,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "A7",
+        "driver": "spi"
+    },
+    "community_layouts": ["65_ansi_blocker_tsangan"],
```

Comment:
named layout is missing the top right most key from the corresponding community layout
```suggestion
```